src/daft-local-execution/src/runtime_stats/mod.rs

```diff
@@ -213,47 +257,6 @@ impl From<&RuntimeStatsContext> for RuntimeStatsEvent {
     }
 }
 
-#[derive(Debug)]
-pub struct RuntimeStats {
```

Comment:
This ended up getting combined with the `render()` function of `RuntimeStatsContext`, which just returns a tuple of name and stringified value. Should be the same as checking the status in between a query and at the end,